pkg/security/security.go

```diff
@@ -231,6 +231,7 @@ type StsRequestParameters struct {
 type Client interface {
 	CSRSign(csrPEM []byte, certValidTTLInSec int64) ([]string, error)
 	Close()
+	GetRootCertBundle() ([]string, error)
```

Comment:
Should the PCDS use this method?

Reply:
No. The secretcache is meant to combine the root certs returned directly from the CA with all the additionally configured TrustAnchors in the system